domain-services/adapter-master-data-service/src/main/java/org/egov/repository/queryBuilder/ProjectQueryBuilder.java

```diff
@@ -31,7 +31,7 @@ public Query buildQuerySearch(ProjectSearchCriteria projectSearchCriteria) {
         }
 
         if (!StringUtils.isEmpty(projectSearchCriteria.getDepartmentEntityId())) {
-            criteria.and("departmentEntityId").is(projectSearchCriteria.getDepartmentEntityId());
+            criteria.and("departmentEntityIds").all(Collections.singletonList(projectSearchCriteria.getDepartmentEntityId()));
```

Comment:
we used to use 'in' operator that can handle anyone or all matching scenario.

Reply:
Done